scripts/travis/install.sh

```diff
@@ -12,12 +12,9 @@ echo "Install Python dependencies"
 ./peep.sh install -r requirements/default.txt
 echo
 
-# Installing dependencies for smoke tests
-if [[ $TEST_SUITE == "smoke" ]]; then
-  virtualenv venv_smoketests
-  source venv_smoketests/bin/activate
-  pip install -r smoketests/requirements.txt
-  deactivate
+# Installing dependencies for UI tests
+if [[ $TEST_SUITE == "ui" ]]; then
+  ./peep.sh install -r requirements/test.txt
```

Comment:
This will potentially prevent us from using `pytest-django` for the UI tests in the future. I suspect this is worth looking into, as it is likely to allow much more control over test setup, including the creation of users and test data.